### ISSUE TITLE
Graphics not displaying

### ISSUE THREAD

**Issue by spatterson82** [nickcam/FlareClusterLayer#23]
I have been trying to use your FlareClusterLayer with a REST service that returns JSON of points.  I am using the GraphicsLayer base class load function to call the REST service and store the point data as the FlareClusterLayer._data.  After that I call the draw function. 

I can see console log statements when I pan around that tell me if the data is clustered or not during the draw function.  I cannot get any graphics to show up on the map.  I am not familiar with what part of your code loads the graphics in the view.  Maybe you have some insight that could help me.  Thank you much.

**Comment by nickcam** [comment]
Hi @spatterson82, no worries. Can you post an example of your current code and problem somewhere - like a gist or plnkr.

**Comment by spatterson82** [comment]
Here is a gist link for the FlareClusterLayer that I added things to.  I commented the functions I added and I also updated the coordinate property names because the JSON I am using has Lat/Long. I will post some functions from the other parts to show how it is being called. 
https://gist.github.com/spatterson82/04c72c9ac7467ce8666f111c51db0259

**Comment by spatterson82** [comment]
Here are the two functions that are part of a main application that build a layer and pass the layers to a Map object. 
https://gist.github.com/spatterson82/4716c810a6a836ee617f6ed2c1a83831

**Comment by nickcam** [comment]
Hi, I can't see anything glaringly wrong with what you're trying without seeing a live demo.
Couple of points though.
- You can just set the xPropertyName and yPropertyName as values in the constructor parameters, don't need to change the default values in the layer itself.
- You could move the calls to get the data outside of the layer and then just call setData() on an instantiated layer object once it's been retrieved. Just to try and keep the layer code black boxed.

For further debugging the drawing occurs in this method FlareClusterLayer.prototype.draw(). 
Add a bunch of console.log calls and/or debug into it. Firstly to confirm it's actually getting hit and then to see how far it's making it through that function. That's where the all of the actual drawing happens so if you're not seeing anything drawn it may be getting short circuited before the drawing occurs.

**Comment by spatterson82** [comment]
Thanks for checking it out.  I will try those suggestions today and try to troubleshoot a bit more.

**Comment by spatterson82** [comment]
@nickcam I finally got the clusters to display on the map today.  The layer visibility is being defaulted to false somewhere.  I don't see that getting set anywhere.  

In the _createCluster function, I added "this.visble = true;" at the very bottom and it works. 

I am going to try and change it now to take a FeatureLayer url to load data from a published map instead of querying live data from a REST endpoint. 

Thanks for taking a look and helping me out.  You put alot of good stuff into the layer.
